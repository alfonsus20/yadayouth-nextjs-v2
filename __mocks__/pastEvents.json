{
    "count": 5,
    "next": null,
    "previous": null,
    "results": [
        {
            "title": "Yada Talks 1.0",
            "date": "2021-05-04T17:14:21Z",
            "text": "Yada Talks is an online mini webinar that aims to educate about and develop the awareness of sustainable development. Yada Talks 1.0 focuses on the topic of self-development and social movements, with Yosephine Devina Wijaya (Key Account Manager at P&G, Top 6 HSBC/HKU Asia Pacific Business Case Competition, Hong Kong) and Yacinta Shafira Pradana (Founder and President of Women Beyond Indonesia, YLI Wave 12) as the speakers.",
            "image": "https://zcspsixwoefhuabatnmr.supabase.co/storage/v1/object/public/images-past-event/event-2.png"
        },
        {
            "title": "Yada Talks 2.0",
            "date": "2021-06-19T18:28:45Z",
            "text": "Yada Talks is an online mini webinar that aims to educate participants on the understanding of sustainable development for young people and further develop their awareness to do so. The topic that Yada Talks 2.0 touched down on was the influence of communication on business growth, with Renata Suyono (Head of Corporate Communication at Decathlon Indonesia) as the speaker.",
            "image": "https://zcspsixwoefhuabatnmr.supabase.co/storage/v1/object/public/images-past-event/event-3.png"
        },
        {
            "title": "Yada Youth Teaching Program",
            "date": "2021-04-24T18:29:34Z",
            "text": "Yada Youth went to Panti Yatim Indonesia for our teaching program that was done on the basis of providing education to children which is our key goal: children empowerment. We taught the children some fun English supplies and got the chance to have personal talks with them. Everyone was more eager to learn and ask!",
            "image": "https://zcspsixwoefhuabatnmr.supabase.co/storage/v1/object/public/images-past-event/event-4.jpg"
        },
        {
            "title": "Yada Talks 3.0",
            "date": "2021-09-19T18:30:11Z",
            "text": "Yada Talks is an online mini webinar that aims to educate about and develop the awareness of sustainable development. Yada Talks 3.0 focuses on the topic of exploring the life and opportunities in campus, with Madeline Agita (Chairwoman IEEE ITB and Operations at Shopee Apprentice Program), Christopher Christian (Head of Kajian Strategis at BEM UI and Medical Executive at Halodoc) and Annisa Apriliani (Vice President at Generasi Baru Indonesia Sulawesi Selatan and awardee of Young leaders for Indonesia National Wave 13 by Mckinsey) as the speakers.",
            "image": "https://zcspsixwoefhuabatnmr.supabase.co/storage/v1/object/public/images-past-event/event-5.png"
        },
        {
            "title": "Money Talk",
            "date": "2021-09-25T18:30:42Z",
            "text": "Yada Youth’s Empowerment Division held a financial literacy workshop in Central Jakarta, precisely at Balai Warga RW 5, Karet Tengsin. The main goal of this workshop is to broaden the participant’s perspective in managing their financial choices, as well as gaining more insight which focuses on discovering more deeply about money management.",
            "image": "https://zcspsixwoefhuabatnmr.supabase.co/storage/v1/object/public/images-past-event/event-6.jpg"
        }
    ]
}