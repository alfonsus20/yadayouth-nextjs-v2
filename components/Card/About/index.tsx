import Image, { StaticImageData } from "next/image";

type Props = {
  image: StaticImageData;
  title: string;
  description: string;
};

const About = ({ image, title, description }: Props) => {
  return (
    <div className="text-center space-y-8">
      <div>
        <h3 className="text-gray-600 font-bold text-3xl mb-2">{title}</h3>
        <div className="h-1 bg-gray-600 w-1/5 mx-auto"></div>
      </div>
      <div>
        <Image
          src={image}
          alt={title}
          placeholder="blur"
          className="rounded-xl"
        />
      </div>
      <p>{description}</p>
    </div>
  );
};

export default About;
