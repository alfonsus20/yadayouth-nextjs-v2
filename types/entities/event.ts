export type Event = {
  title: string;
  date: string;
  text: string;
  image: string;
};
