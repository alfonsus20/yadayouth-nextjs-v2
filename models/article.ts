import { AxiosPromise } from "axios";
import api from "../api";
import { APIResponsePaginated } from "../types/apiResponse";
import { Article } from "../types/entities/article";

export const fetchArticles = (): APIResponsePaginated<Article> => {
  return api.get("/article");
};

export const fetchArticlesById = (id : string): AxiosPromise<Article> => {
  return api.get(`/article/${id}`);
};
