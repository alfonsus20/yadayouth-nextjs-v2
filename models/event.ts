import api from "../api";
import { APIResponsePaginated } from "../types/apiResponse";
import { Event } from "../types/entities/event";

export const fetchPastEvents = (): APIResponsePaginated<Event> => {
  return api.get("/past-event");
};
