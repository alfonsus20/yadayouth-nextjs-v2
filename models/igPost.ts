import { igApi } from "../api";
import { APIResponse } from "../types/apiResponse";
import { IGPost } from "../types/entities/igPost";

export const fetchIGPosts = (): APIResponse<Array<IGPost>> => {
  return igApi.get("/");
};
