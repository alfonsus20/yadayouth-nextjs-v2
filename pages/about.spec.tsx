import { render } from '@testing-library/react';
import * as svc from '../models/event';
import pastEvents from '../__mocks__/pastEvents.json';
import About, { getStaticProps } from './about.page';

jest.mock('next/image', () => ({
  __esModule: true,
  default: (props: any) => {
    // eslint-disable-next-line @next/next/no-img-element
    return <img {...props} alt="image" />;
  },
}));

describe('About page unit test', () => {
  it('should match snapshot', async () => {
    jest.spyOn(svc, 'fetchPastEvents').mockResolvedValue({
      data: pastEvents,
      status: 200,
      statusText: 'Ok',
      headers: {},
      config: {},
    });
    await getStaticProps({});
    const container = render(<About data={pastEvents.results} />);
    expect(container).toMatchSnapshot();
  });
});
