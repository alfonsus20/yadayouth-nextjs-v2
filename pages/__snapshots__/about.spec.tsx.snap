// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page unit test should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <section
          class="relative flex justify-center items-center"
          style="min-height: calc(100vh - 80px);"
        >
          <div
            class="max-w-screen-lg px-16 text-center text-white min-h-[400px] flex items-center"
          >
            <div
              class="relative z-10 pt-8 pb-20"
            >
              <h1
                class="text-3xl xxs:text-4xl sm:text-5xl font-bold"
              >
                TENTANG KAMI
              </h1>
              <div
                class="bg-blue h-2 w-1/3 my-4 mx-auto rounded-md"
              />
              <p
                class="text-lg mb-4"
              >
                Yada Youth adalah organisasi nirlaba yang bertujuan untuk fokus pada pemberdayaan anak-anak dan remaja Indonesia yang hidup dalam kemiskinan melalui upaya kolektif untuk meminimalkan faktor sistemik yang menyebabkan kemiskinan.
              </p>
              <button
                class="hover:opacity-90 transition-opacity disabled:opacity-75 disabled:cursor-not-allowed py-2 flex items-center mx-auto rounded-full bg-blue text-white px-10 text-lg"
                data-testid="button"
                href="/#selengkapnya"
              >
                SELENGKAPNYA
              </button>
            </div>
          </div>
          <img
            alt="image"
            layout="fill"
            objectfit="cover"
            objectposition="top"
            placeholder="blur"
            src="about.jpg"
          />
          <div
            class="absolute top-0 left-0 bottom-0 right-0 bg-neutral-900 opacity-50"
          />
        </section>
        <section
          class="max-w-screen-xl mx-auto px-8 sm:px-16 py-16 grid sm:grid-cols-2 gap-10"
          id="selengkapnya"
        >
          <div
            class="text-center space-y-8"
          >
            <div>
              <h3
                class="text-gray-600 font-bold text-3xl mb-2"
                data-testid="card-title"
              >
                VISI KAMI
              </h3>
              <div
                class="h-1 bg-gray-600 w-1/5 mx-auto"
              />
            </div>
            <div>
              <img
                alt="image"
                class="rounded-xl"
                data-testid="card-image"
                placeholder="blur"
                src="vision.jpg"
              />
            </div>
            <p
              data-testid="card-description"
            >
              Membangun masyarakat yang cocok untuk masa depan umat manusia, di mana anak-anak, remaja, dan pemuda diberi makan dengan baik, berpendidikan baik, sehat, setara, dilindungi, dan dirawat.
            </p>
          </div>
          <div
            class="text-center space-y-8"
          >
            <div>
              <h3
                class="text-gray-600 font-bold text-3xl mb-2"
                data-testid="card-title"
              >
                MISI KAMI
              </h3>
              <div
                class="h-1 bg-gray-600 w-1/5 mx-auto"
              />
            </div>
            <div>
              <img
                alt="image"
                class="rounded-xl"
                data-testid="card-image"
                placeholder="blur"
                src="mission.jpg"
              />
            </div>
            <p
              data-testid="card-description"
            >
              Terus meningkatkan standar hidup anak-anak, remaja, dan remaja untuk membantu mereka hidup lebih bahagia, memberdayakan mereka untuk mencapai lebih banyak, dan berkontribusi lebih banyak kepada masyarakat
            </p>
          </div>
        </section>
        <section
          class="bg-gradient-to-b from-[#E5AB1A] rounded-t-[80px]"
        >
          <div
            class="max-w-screen-lg mx-auto px-8 xs:px-16 py-16"
          >
            <h3
              class="text-white text-3xl xs:text-4xl font-bold text-center mb-16 xs:mb-28 px-8"
            >
              Behind Yada Youth
            </h3>
            <div
              class="relative"
            >
              <div
                class="absolute top-1/2 transform -translate-y-1/2 right-0 w-20 h-[75%] hidden lg:flex flex-col justify-between items-center"
              >
                <div
                  class="bg-orange h-12 w-12 rounded-full"
                />
                <div
                  class="bg-orange h-12 w-12 rounded-full mb-28"
                />
                <div
                  class="bg-orange h-12 w-12 rounded-full"
                />
                <div
                  class="w-4 absolute bg-orange left-1/2 top-1/2 transform -translate-y-1/2 -translate-x-1/2 h-[98%]"
                />
              </div>
              <div
                class="bg-[#686DEC] text-white p-8 rounded-2xl relative max-w-2xl mx-auto mb-8 xs:mb-20"
              >
                <div
                  class="rounded-full border-4 border-white w-64 h-64 xs:w-48 xs:h-48 overflow-hidden xs:absolute xs:-top-20 xs:-left-12 mx-auto"
                >
                  <img
                    alt="image"
                    height="400"
                    placeholder="blur"
                    src="hanif.png"
                    width="400"
                  />
                </div>
                <h2
                  class="font-bold text-3xl xs:text-4xl my-4 xs:mt-6 xs:mb-8 xs:ml-32"
                >
                  Hanif
                </h2>
                <p
                  class="text-justify"
                >
                  Hanif adalah pendiri dan direktur Yada Youth Indonesia. Melalui kehidupan kampusnya, dia memiliki latar belakang akademis yang kuat dan sering terlibat dalam kegiatan ekstrakurikuler. Yang membuatnya mendapat predikat Mahasiswa Berprestasi Ketiga di universitasnya, Institut Teknologi Sepuluh Nopember (ITS). Selain itu, ia juga sangat bersemangat dalam berwirausaha, yang membawanya untuk ikut mendirikan PT. Grup Milenial Global. Hanif memiliki komitmen jangka panjang untuk mengatasi kemiskinan, melihat sebagian besar karya ilmiah dan inovasinya berfokus pada penyelesaian masalah keuangan, ekonomi, dan sosial di pedesaan. Kini, didorong oleh mimpi masa kecilnya, ia bertekad untuk memberdayakan anak-anak miskin di seluruh Indonesia melalui Yada Youth Indonesia.
                </p>
              </div>
              <div
                class="bg-[#686DEC] text-white p-8 rounded-2xl relative max-w-2xl mx-auto flex flex-col md:flex-row items-center gap-8 mb-12 xs:mb-36"
              >
                <div
                  class="rounded-full overflow-hidden w-64 h-64 md:w-56 md:h-56 md:flex-shrink-0 flex justify-center"
                >
                  <img
                    alt="image"
                    height="400"
                    placeholder="blur"
                    src="logo.png"
                    width="400"
                  />
                </div>
                <div>
                  <h2
                    class="font-bold text-3xl xs:text-4xl mb-4"
                  >
                    YadaYouth.id
                  </h2>
                  <p
                    class="text-justify"
                  >
                    19 Februari 2021 YadaYouth.id di bangun oleh dua anak muda yaitu Hanif dan Stephani.
                  </p>
                </div>
              </div>
              <div
                class="bg-[#686DEC] text-white p-8 rounded-2xl relative max-w-2xl mx-auto"
              >
                <div
                  class="rounded-full border-4 border-white w-64 h-64 xs:w-48 xs:h-48 overflow-hidden xs:absolute xs:-top-20 xs:-left-12 mx-auto"
                >
                  <img
                    alt="image"
                    height="250"
                    placeholder="blur"
                    src="stephani.png"
                    width="250"
                  />
                </div>
                <h2
                  class="font-bold text-3xl xs:text-4xl my-4 xs:mt-6 xs:mb-8 xs:ml-32"
                >
                  Stephani
                </h2>
                <p
                  class="text-justify"
                >
                  Stephani adalah wakil direktur Yada Youth Indonesia yang saat ini menjadi mahasiswa di Universitas Gadjah Mada. Sepanjang sekolah menengah, ia dikenal sebagai siswa aktif dengan prestasi akademik yang baik, lulus dengan IPK 3,8 dari 4,0. Dia sangat tergila-gila dengan konektivitas manusia dalam organisasi sosial. Upaya Stephani dalam advokasi untuk Anak dimulai pada tahun 2019 ketika ia menjabat sebagai penggalangan dana di UNICEF Indonesia untuk kampanye penyediaan sanitasi dan pendidikan untuk Anak Papua. Ia kemudian melanjutkan perannya sebagai relawan pembangunan di Panti Asuhan Karunia, sebuah panti asuhan yang dibangun khusus untuk Yatim Papua di Surabaya. Stephani berjuang untuk keyakinan bahwa anak-anak tanpa memandang latar belakang harus diberikan kesempatan yang sama seperti mereka yang lahir dalam situasi yang lebih istimewa dan saat ini berjuang untuk tujuan melalui Yada Youth Indonesia.
                </p>
              </div>
            </div>
          </div>
        </section>
        <section
          class="bg-gradient-to-b from-[#686DEC] rounded-t-[80px]"
        >
          <div
            class="py-16"
          >
            <h3
              class="text-white text-3xl xs:text-4xl font-bold text-center mb-12 px-8"
            >
              Program yang Pernah Dilakukan
            </h3>
            <div
              class="overflow-x-hidden sm:py-16"
            >
              <div
                class="transform sm:scale-150"
              >
                <div
                  class="react-multi-carousel-list  "
                  dir="ltr"
                >
                  <ul
                    class="react-multi-carousel-track "
                    style="transition: none; overflow: unset; transform: translate3d(0px,0,0);"
                  />
                </div>
              </div>
            </div>
          </div>
        </section>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <section
        class="relative flex justify-center items-center"
        style="min-height: calc(100vh - 80px);"
      >
        <div
          class="max-w-screen-lg px-16 text-center text-white min-h-[400px] flex items-center"
        >
          <div
            class="relative z-10 pt-8 pb-20"
          >
            <h1
              class="text-3xl xxs:text-4xl sm:text-5xl font-bold"
            >
              TENTANG KAMI
            </h1>
            <div
              class="bg-blue h-2 w-1/3 my-4 mx-auto rounded-md"
            />
            <p
              class="text-lg mb-4"
            >
              Yada Youth adalah organisasi nirlaba yang bertujuan untuk fokus pada pemberdayaan anak-anak dan remaja Indonesia yang hidup dalam kemiskinan melalui upaya kolektif untuk meminimalkan faktor sistemik yang menyebabkan kemiskinan.
            </p>
            <button
              class="hover:opacity-90 transition-opacity disabled:opacity-75 disabled:cursor-not-allowed py-2 flex items-center mx-auto rounded-full bg-blue text-white px-10 text-lg"
              data-testid="button"
              href="/#selengkapnya"
            >
              SELENGKAPNYA
            </button>
          </div>
        </div>
        <img
          alt="image"
          layout="fill"
          objectfit="cover"
          objectposition="top"
          placeholder="blur"
          src="about.jpg"
        />
        <div
          class="absolute top-0 left-0 bottom-0 right-0 bg-neutral-900 opacity-50"
        />
      </section>
      <section
        class="max-w-screen-xl mx-auto px-8 sm:px-16 py-16 grid sm:grid-cols-2 gap-10"
        id="selengkapnya"
      >
        <div
          class="text-center space-y-8"
        >
          <div>
            <h3
              class="text-gray-600 font-bold text-3xl mb-2"
              data-testid="card-title"
            >
              VISI KAMI
            </h3>
            <div
              class="h-1 bg-gray-600 w-1/5 mx-auto"
            />
          </div>
          <div>
            <img
              alt="image"
              class="rounded-xl"
              data-testid="card-image"
              placeholder="blur"
              src="vision.jpg"
            />
          </div>
          <p
            data-testid="card-description"
          >
            Membangun masyarakat yang cocok untuk masa depan umat manusia, di mana anak-anak, remaja, dan pemuda diberi makan dengan baik, berpendidikan baik, sehat, setara, dilindungi, dan dirawat.
          </p>
        </div>
        <div
          class="text-center space-y-8"
        >
          <div>
            <h3
              class="text-gray-600 font-bold text-3xl mb-2"
              data-testid="card-title"
            >
              MISI KAMI
            </h3>
            <div
              class="h-1 bg-gray-600 w-1/5 mx-auto"
            />
          </div>
          <div>
            <img
              alt="image"
              class="rounded-xl"
              data-testid="card-image"
              placeholder="blur"
              src="mission.jpg"
            />
          </div>
          <p
            data-testid="card-description"
          >
            Terus meningkatkan standar hidup anak-anak, remaja, dan remaja untuk membantu mereka hidup lebih bahagia, memberdayakan mereka untuk mencapai lebih banyak, dan berkontribusi lebih banyak kepada masyarakat
          </p>
        </div>
      </section>
      <section
        class="bg-gradient-to-b from-[#E5AB1A] rounded-t-[80px]"
      >
        <div
          class="max-w-screen-lg mx-auto px-8 xs:px-16 py-16"
        >
          <h3
            class="text-white text-3xl xs:text-4xl font-bold text-center mb-16 xs:mb-28 px-8"
          >
            Behind Yada Youth
          </h3>
          <div
            class="relative"
          >
            <div
              class="absolute top-1/2 transform -translate-y-1/2 right-0 w-20 h-[75%] hidden lg:flex flex-col justify-between items-center"
            >
              <div
                class="bg-orange h-12 w-12 rounded-full"
              />
              <div
                class="bg-orange h-12 w-12 rounded-full mb-28"
              />
              <div
                class="bg-orange h-12 w-12 rounded-full"
              />
              <div
                class="w-4 absolute bg-orange left-1/2 top-1/2 transform -translate-y-1/2 -translate-x-1/2 h-[98%]"
              />
            </div>
            <div
              class="bg-[#686DEC] text-white p-8 rounded-2xl relative max-w-2xl mx-auto mb-8 xs:mb-20"
            >
              <div
                class="rounded-full border-4 border-white w-64 h-64 xs:w-48 xs:h-48 overflow-hidden xs:absolute xs:-top-20 xs:-left-12 mx-auto"
              >
                <img
                  alt="image"
                  height="400"
                  placeholder="blur"
                  src="hanif.png"
                  width="400"
                />
              </div>
              <h2
                class="font-bold text-3xl xs:text-4xl my-4 xs:mt-6 xs:mb-8 xs:ml-32"
              >
                Hanif
              </h2>
              <p
                class="text-justify"
              >
                Hanif adalah pendiri dan direktur Yada Youth Indonesia. Melalui kehidupan kampusnya, dia memiliki latar belakang akademis yang kuat dan sering terlibat dalam kegiatan ekstrakurikuler. Yang membuatnya mendapat predikat Mahasiswa Berprestasi Ketiga di universitasnya, Institut Teknologi Sepuluh Nopember (ITS). Selain itu, ia juga sangat bersemangat dalam berwirausaha, yang membawanya untuk ikut mendirikan PT. Grup Milenial Global. Hanif memiliki komitmen jangka panjang untuk mengatasi kemiskinan, melihat sebagian besar karya ilmiah dan inovasinya berfokus pada penyelesaian masalah keuangan, ekonomi, dan sosial di pedesaan. Kini, didorong oleh mimpi masa kecilnya, ia bertekad untuk memberdayakan anak-anak miskin di seluruh Indonesia melalui Yada Youth Indonesia.
              </p>
            </div>
            <div
              class="bg-[#686DEC] text-white p-8 rounded-2xl relative max-w-2xl mx-auto flex flex-col md:flex-row items-center gap-8 mb-12 xs:mb-36"
            >
              <div
                class="rounded-full overflow-hidden w-64 h-64 md:w-56 md:h-56 md:flex-shrink-0 flex justify-center"
              >
                <img
                  alt="image"
                  height="400"
                  placeholder="blur"
                  src="logo.png"
                  width="400"
                />
              </div>
              <div>
                <h2
                  class="font-bold text-3xl xs:text-4xl mb-4"
                >
                  YadaYouth.id
                </h2>
                <p
                  class="text-justify"
                >
                  19 Februari 2021 YadaYouth.id di bangun oleh dua anak muda yaitu Hanif dan Stephani.
                </p>
              </div>
            </div>
            <div
              class="bg-[#686DEC] text-white p-8 rounded-2xl relative max-w-2xl mx-auto"
            >
              <div
                class="rounded-full border-4 border-white w-64 h-64 xs:w-48 xs:h-48 overflow-hidden xs:absolute xs:-top-20 xs:-left-12 mx-auto"
              >
                <img
                  alt="image"
                  height="250"
                  placeholder="blur"
                  src="stephani.png"
                  width="250"
                />
              </div>
              <h2
                class="font-bold text-3xl xs:text-4xl my-4 xs:mt-6 xs:mb-8 xs:ml-32"
              >
                Stephani
              </h2>
              <p
                class="text-justify"
              >
                Stephani adalah wakil direktur Yada Youth Indonesia yang saat ini menjadi mahasiswa di Universitas Gadjah Mada. Sepanjang sekolah menengah, ia dikenal sebagai siswa aktif dengan prestasi akademik yang baik, lulus dengan IPK 3,8 dari 4,0. Dia sangat tergila-gila dengan konektivitas manusia dalam organisasi sosial. Upaya Stephani dalam advokasi untuk Anak dimulai pada tahun 2019 ketika ia menjabat sebagai penggalangan dana di UNICEF Indonesia untuk kampanye penyediaan sanitasi dan pendidikan untuk Anak Papua. Ia kemudian melanjutkan perannya sebagai relawan pembangunan di Panti Asuhan Karunia, sebuah panti asuhan yang dibangun khusus untuk Yatim Papua di Surabaya. Stephani berjuang untuk keyakinan bahwa anak-anak tanpa memandang latar belakang harus diberikan kesempatan yang sama seperti mereka yang lahir dalam situasi yang lebih istimewa dan saat ini berjuang untuk tujuan melalui Yada Youth Indonesia.
              </p>
            </div>
          </div>
        </div>
      </section>
      <section
        class="bg-gradient-to-b from-[#686DEC] rounded-t-[80px]"
      >
        <div
          class="py-16"
        >
          <h3
            class="text-white text-3xl xs:text-4xl font-bold text-center mb-12 px-8"
          >
            Program yang Pernah Dilakukan
          </h3>
          <div
            class="overflow-x-hidden sm:py-16"
          >
            <div
              class="transform sm:scale-150"
            >
              <div
                class="react-multi-carousel-list  "
                dir="ltr"
              >
                <ul
                  class="react-multi-carousel-track "
                  style="transition: none; overflow: unset; transform: translate3d(0px,0,0);"
                />
              </div>
            </div>
          </div>
        </div>
      </section>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
